contracts/contracts/harvest/FixedRateDripper.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IVault } from "../interfaces/IVault.sol";
+import { Dripper } from "./Dripper.sol";
+
+/**
+ * @title Fixed Rate Dripper
+ *
+ * Similar to the Dripper, Fixed Rate Dripper drips out yield per second.
+ * However the Strategist decides the rate and it doesn't change after
+ * a drip.
+ *
+ */
+
+contract FixedRateDripper is Dripper {
+    using SafeERC20 for IERC20;
+
+    event DripRateUpdated(uint192 oldDripRate, uint192 newDripRate);
+
+    /**
+     * @dev Verifies that the caller is the Governor or Strategist.
+     */
+    modifier onlyGovernorOrStrategist() {
+        require(
+            isGovernor() || msg.sender == IVault(vault).strategistAddr(),
+            "Caller is not the Strategist or Governor"
+        );
+        _;
+    }
+
+    constructor(address _vault, address _token) Dripper(_vault, _token) {}
+
+    /// @inheritdoc Dripper
+    function setDripDuration(uint256) external virtual override {
+        // Not used in FixedRateDripper
+        revert("Drip duration disabled");
+    }
+
+    /// @inheritdoc Dripper
+    function _collect() internal virtual override {
+        // Calculate amount to send
+        uint256 balance = IERC20(token).balanceOf(address(this));
+        uint256 amountToSend = _availableFunds(balance, drip);
+
+        // Update timestamp
+        drip.lastCollect = uint64(block.timestamp);
+
+        // Send funds
+        IERC20(token).safeTransfer(vault, amountToSend);
+    }
+
+    /**
+     * @dev Sets the drip rate. Callable by Strategist or Governor.
+     *      Can be set to zero to stop dripper.
+     * @param _perSecond Rate of WETH to drip per second
+     */
+    function setDripRate(uint192 _perSecond) external onlyGovernorOrStrategist {
+        emit DripRateUpdated(_perSecond, drip.perSecond);
+
+        // Collect at existing rate
+        _collect();
```

Comment:
Maybe add a comment here as it is super important that the `_collect` be called before the `drip.perSecond` is set. And the reason is: 
- the old harvester would not have the drip.perSecond populated by an external function. In order for the dripper to be "initialised" the first call to `collect` would drip no funds since drip's `lastCollect` and `perSecond` are both zero. Meaning the available funds calculation `uint256 allowed = elapsed * _drip.perSecond` would have a very big `elapsed` number and 0 for `perSecond` number returning 0 on the first call. And initializing the drip storage value to recent `lastCollect` and some value for `perSecond`
- the new harvester directly initializes the `drip.perSecond` and if it were to do that before the `_collect()` call is made the `uint256 allowed = elapsed * _drip.perSecond` would result in a very big number. Meaning that if we were to deploy the dripper and first send funds to it, and then call the `setDripRate` all of the funds would be immediately sent to the Vault. 
- but the function correctly calls `_collect` ensuring that the `lastCollect` is not 0 but a currentBlock number. Preventing the described potential problem.

Reply:
Yep, have added the comment. Thanks